packages/@pollyjs/core/src/persisters/rest/ajax.js

```diff
@@ -1,6 +1,7 @@
 const { keys } = Object;
-const REQUEST_ASYNC = !/PhantomJS/.test(self.navigator.userAgent);
-const NativeXMLHttpRequest = self.XMLHttpRequest;
+const REQUEST_ASYNC =
```

Comment:
Assumes whatever polyfill being used for XHR in node would support the nuance being worked around within phantom.